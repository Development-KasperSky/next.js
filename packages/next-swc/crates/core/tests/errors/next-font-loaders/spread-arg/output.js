import inter from "@next/font/google?Inter;{};[]";
const a = fn(...{}, ...[]);
