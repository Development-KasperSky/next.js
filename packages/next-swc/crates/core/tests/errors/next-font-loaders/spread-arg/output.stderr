
  x Font loaders don't accept spreads
   ,-[input.js:4:1]
 4 | const inter = Inter(...{}, ...[])
   :                     ^^^
   `----

  x Font loaders don't accept spreads
   ,-[input.js:4:1]
 4 | const inter = Inter(...{}, ...[])
   :                            ^^^
   `----
